handbook/business-operations/README.md

```diff
@@ -334,16 +334,17 @@ At Fleet, everyone is recognized on their [workiversary](https://fleetdm.com/han
 4. Use the teammate's information to benchmark in [Pave](https://www.pave.com/) (login details in 1Password). You can pattern match from previous benchmarking entries, and include all company assumtions. Add the direct link to the Pave benchmark.
 
 
-### Update a team member's compensation
-To [change a team member's compensation](https://fleetdm.com/handbook/company/communications#compensation-changes), follow these steps:
-1. Create a copy of the ["Values assessment" template](https://docs.google.com/spreadsheets/d/1P5TyRV2v-YN0aR_X8vd8GksKcr3uHfUDdshqpVzamV8/edit?usp=drive_link) and move it to the team member's [personnel folder in Google Drive](https://drive.google.com/drive/folders/1UL7o3BzkTKnpvIS4hm_RtbOilSABo3oG?usp=drive_link).
+### Update a teammate's compensation
```

Comment:
I suggest not changing the title here or adding a stub to the bottom of the page (See https://fleetdm.com/handbook/business-operations#stubs) so we don't break links.